frontend/src/pages/ProfileUpdate.tsx

```diff
@@ -23,6 +23,9 @@ export function ProfileUpdate() {
   ) => {
     event.preventDefault();
     await updateUserProfile();
+    setTimeout(() => {
+      navigate("/login");
+    }, 1000);
```

Comment:
Not sure if this should be 1000. Some place else @carlos-ponton wants to add 2000. Should it be the same if yes, can we create a global function to reuse?